media/js/base/fxa-utm-referral.js

```diff
@@ -78,24 +75,29 @@
         }
 
         for (var i = 0; i < ctaLinks.length; i++) {
-            // get the current FxA links.
+            // get the link off the element
             var oldAccountsLink = ctaLinks[i].href;
```

Comment:
Would adding a check that the link `href` is not undefined here be worthwhile, to avoid the script throwing an error should someone accidentally add the class to another type of element?

Reply:
You mean like I did? 😅 

✅